R/helpers.R

```diff
@@ -680,9 +680,9 @@ h_is_positive_definite <- function(x, tolerance = 1e-08) {
 #'
 #' @export
 #'
-h_get_slots <- function(object, names) {
+h_slots <- function(object, names) {
   assert_true(isS4(object))
-  assert_character(names, min.len = 1, any.missing = FALSE)
+  assert_character(names, min.len = 1, any.missing = FALSE, null.ok = TRUE)
```

Comment:
This is ok, as it works very well with e.g.: `inits <- do.call(model@init, h_slots(data, formalArgs(model@init)))` in `mcmc.R` file, in case of no formal arguments in `model@init`.

Reply:
hm ok good